InnowiseSQL.sql

```diff
@@ -1 +1,406 @@
-﻿
\ No newline at end of file
+﻿--CREATING AND INSERTING
+CREATE DATABASE BankList
+GO
+
+use BankList;
+
+CREATE TABLE Banks
+(
+BankID INT PRIMARY KEY IDENTITY,
+BankName NVARCHAR(15) NOT NULL
+)
+
+CREATE TABLE Filials
+(
+FilialID INT PRIMARY KEY IDENTITY,
+Adress NVARCHAR(30) NOT NULL,
+Bnk_ID INT,
+CONSTRAINT FK_Filial_To_Bank FOREIGN KEY (Bnk_ID) REFERENCES Banks (BankID) ON DELETE CASCADE
+)
+
+CREATE TABLE Towns
+(
+TownID INT PRIMARY KEY IDENTITY,
+TownName NVARCHAR(15) NOT NULL
+)
+
+CREATE TABLE Buildings
+(
+Fil_ID INT,
+Twn_ID INT,
+CONSTRAINT FK_Building_to_Filial FOREIGN KEY (Fil_ID) REFERENCES Filials (FilialID) ON DELETE CASCADE,
+CONSTRAINT FK_Building_To_Town FOREIGN KEY (Twn_ID) REFERENCES Towns (TownID) ON DELETE CASCADE
+)
+
+CREATE TABLE SocStatus
+(
+SocStatusID INT PRIMARY KEY IDENTITY,
+Name NVARCHAR(15) NOT NULL
+)
+
+CREATE TABLE Clients
+(
+ClientID INT PRIMARY KEY IDENTITY,
+Name NVARCHAR(20) NOT NULL,
+Surname NVARCHAR(30) NOT NULL,
+Patronymic NVARCHAR(30),
+Gender NVARCHAR(3) NOT NULL,
+SocSt_ID INT,
+CONSTRAINT FK_Clients_To_SocStatus FOREIGN KEY(SocSt_ID) REFERENCES SocStatus(SocStatusID)
+)
+
+CREATE TABLE Accounts
+(
+AccountID INT PRIMARY KEY IDENTITY,
+Bnk_ID INT,
+Cl_ID INT,
+Balance MONEY DEFAULT 0.0,
+CONSTRAINT FK_Account_To_Bank FOREIGN KEY (Bnk_ID) REFERENCES Banks (BankID) ON DELETE CASCADE,
+CONSTRAINT FK_Account_To_Client FOREIGN KEY (Cl_ID) REFERENCES Clients (ClientID) ON DELETE CASCADE,
+CONSTRAINT UQ_BankID_ClientID UNIQUE (Bnk_ID, Cl_ID)
+)
+
+CREATE TABLE Cards
+(
+CardID INT PRIMARY KEY IDENTITY,
+Balance MONEY DEFAULT 0.0,
+Ac_ID INT,
+CONSTRAINT FK_Cards_To_Account FOREIGN KEY (Ac_ID) REFERENCES Accounts (AccountID) ON DELETE CASCADE
+)
+
+
+
+INSERT INTO Towns (TownName)
+VALUES  ('Минск'),
```

Comment:
Find a way to fill in tables with Russian data correctly
Right now after filling tables with data you will see a lot of ??????????????????????
![image](https://user-images.githubusercontent.com/33979910/161213604-6c27b370-9593-45bf-ac42-f37eb9d36a59.png)